src/utils/registry.ts

```diff
@@ -62,6 +62,21 @@ export const getInstallCmd = async (components: string[]) => {
   return `npx shadcn-ui@latest add ${componentStr}`;
 };
 
+
+export const getInstallMultipleComponents = async (component: string) => {
```

Comment:
Hi @selemondev

The changes look good to me. Can you just update the name of this function from ```getInstallMultipleComponents``` to ```getInstallMultipleComponentsCmd```? I think it should be clear that this function return a command.